backend/fms_core/template_importer/_utils.py

```diff
@@ -14,8 +14,8 @@ def input_to_date_and_none(date_input):
     elif isinstance(date_input, datetime.date):
         return date_input
     else:
-        year, month, day = date_input.split("-")
         try:
+            year, month, day = date_input.split("-")
```

Comment:
eheh yeah this would crash ... Good idea to pair this with an additional detail in the error message.